.github/CONTRIBUTING.md

```diff
@@ -177,6 +177,25 @@ lintr::lint_package()
 
 [styler](https://CRAN.R-project.org/package=styler) will not fix all linting issues, so we recommend using that first, then using [lintr](https://lintr.r-lib.org/articles/lintr.html) to check for places you may need to manually fix styling issues such as line length or not using snake_case.
 
+## Folder and script structure conventions
+
+In an R package you are not allowed to have any sub directories in the R/ folder. Where possible we should have:
+
+* one script per function 
+or 
+* one script per function family if a function belongs to a family
+
+The script should share the name of the function or family. If needed then a <family>_utils.R script should be used for 
+internal only functions that relate to a specific family.
+
+Documentation for all data shipped with the packages is kept in `R/datasets_documentation.R`. Scripts used for preparing 
+data used in the package is not in the R folder, it is in the `data-raw/` folder, helper functions for this can be found 
+in the `R/datasets_utils.R` folder.
```

Comment:
"... and notes on how to run these scripts are included further down in this document."

Reply:
added